app/shared/new-relic.ts

```diff
@@ -0,0 +1,44 @@
+'use client'
+import { BrowserAgent } from '@newrelic/browser-agent/loaders/browser-agent'
+import { useUser } from './hooks/useUser'
+import { useEffect } from 'react'
+import { useCampaign } from './hooks/useCampaign'
+import { usePathname } from 'next/navigation'
+import { isProductRoute } from './utils/isProductRoute'
+
+export const newrelic = new BrowserAgent({
+  info: {
+    licenseKey: 'NRJS-f5250be4de5180494b6',
+    applicationID: '1120481134',
+  },
```

Comment:
For now, I didn't bother separating dev/qa/prod. I was going to just work on separating / disabling them within newrelic itself, or here via path checks if it gets problematic.